libkbfs/init.go

```diff
@@ -431,7 +447,18 @@ func Init(ctx Context, params InitParams, keybaseServiceCn KeybaseServiceCn,
 
 func doInit(ctx Context, params InitParams, keybaseServiceCn KeybaseServiceCn,
 	log logger.Logger) (Config, error) {
-	config := NewConfigLocal(func(module string) logger.Logger {
+	mode := InitDefault
+	switch params.Mode {
+	case InitDefaultString:
+		// Already the default
```

Comment:
Let's put a comment in here too, so we can search for `Initializing in.*mode` in logs.